content/pt-br/examples/service/networking/example-ingress.yaml

```
@@ -2,8 +2,6 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: example-ingress
-  annotations:
-    nginx.ingress.kubernetes.io/rewrite-target: /$1
 spec:
```

```suggestion
spec:
  ingressClassName: nginx
```
As per comments of #47239, same changes will be required in this one too.

---

Thanks, but no need, already in the main branch, no need for rebase
https://github.com/kubernetes/website/pull/47168